src/errorDictionary.js

```diff
@@ -0,0 +1,63 @@
+import fs from "fs";
+
+let dictionary;
+
+module.exports = {
+  init: function (config) {
+    if(!dictionary && config){
+
+      // check if file
+      if(typeof(config) === 'string' && fs.existsSync(config)){
+        try{
+          dictionary = JSON.parse(fs.readFileSync(config, 'utf8'));
+        }catch(e){
+          console.warn("Error loading error dictionary from file " + config + ". " + JSON.stringify(e));
```

Comment:
import `logger` from "./util/logger" and use `logger` instead of `console` in this file